src/components/Invite.js

```diff
@@ -33,6 +34,11 @@ function Invite({ toggleInvite }) {
     navigator.clipboard.writeText(shareUrl);
   }
 
+  async function reloadPage() {
+    await delay(500);
```

Comment:
I think we should remove this delay.

Reply:
As an future optimization, we could actually optionally reload the page? For example, we only reload when we know for sure the badge has been awarded. We could probably have some local variables to keep track of the state. 
We could even add the badge locally w/o doing a full reload and fetching from the firestore.

Reply:
Delay is removed in updated PR. I agree in regards to tracking state or something. I had started looking into this to optimize how we reload or preferably only update the necessary piece(s)